projects/graph/src/graph.py

```diff
@@ -3,7 +3,36 @@
 """
 
 
+class Vertex:
+    def __init__(self, vertex):
+        self.vertex = vertex
+        self.edges = []
```

Comment:
Would it make sense to use a set instead of a list to prevent adding an edge that is already there?